bin/annotation_workflow.py

```diff
@@ -0,0 +1,76 @@
+import json, re
+from pathlib import Path
+
+def github_action_notice(file, url, line):
+    """
+    Print GitHub action notice with file path, URL, and line number.
+
+    Parameters:
+        file (str): File path.
```

Comment:
Good you added these! Though in modern Python they go in the signature, so here:

`github_action_notice(file: str, url: str, line: int)`

(though probably `file` should be a `Path`)